js/Glacier2/callback/timeUtil.ts

```diff
@@ -0,0 +1,8 @@
+const millisecondsBeforeEpoch = 719162n * 24n * 60n * 60n * 1000n; // daysBeforeEpoch converted to milliseconds
```

Comment:
Should we make put this in a common top level folder for use by future demos?

Reply:
We already use it in a few demos. I will refactor this in a follow up PR.